clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/cache/Keys.groovy

```diff
@@ -69,8 +69,9 @@ class Keys {
         result << [
             application: parts[3],
             account: parts[2],
-            name: parts[4],
-            cluster: parts[4],
+            name: parts[5],
+            cluster: parts[5],
+            category: parts[4],
```

Comment:
`category` is pretty generic. I wouldn't know what the heck it meant upon first glance. `clusterType`? `jobType`? `podOwner`? `clusterF$%`?